package.json

```diff
@@ -1,78 +1,52 @@
 {
-  "name": "@ngui/auto-complete",
-  "version": "2.0.0",
-  "description": "Angular Input Autocomplete",
-  "license": "MIT",
```

Comment:
Why these lines removed?

Reply:
`/auto-complete/projects/auto-complete/package.json`

Library `package.json` file should be here. This `package.json` is for the base project.

Reply:
It should be fixed. But refer - `auto-complate/projects/auto-complete/package.json` file.